seongnam/controllers/DataNumberController.js

```diff
@@ -0,0 +1,16 @@
+//controller/dataController.js
+
+const dataService = require('../services/fixDataService');
+const fixDatas2 = async(req,res)=>{
```

Comment:
함수이름은 정확히 어떤 행위를 하는지 나타내 주세요.
```suggestion
const fixUser = async(req,res)=>{
```